tests/dragonfly/replication_test.py

```diff
@@ -918,6 +918,8 @@ async def test_role_command(df_local_factory, n_keys=20):
     await c_replica.execute_command(f"REPLICAOF localhost {master.port}")
     await wait_available_async(c_replica)
 
+    await asyncio.sleep(1)
```

Comment:
That makes sense. The only concern I have, is that maybe, if we stress the CI enough (by having multiple jobs from separate PR's) this `1s` might not be enough? 

I think it suffices for now so I will accept but if it ever fails again we would need a better solution

Reply:
Indeed, I really hate adding sleeps for tests to pass. It's unstable. But that's our current state of the testing framework :/

Reply:
> But that's our current state of the testing framework 
Yes and at some point we should find a longterm solution :)

Reply:
I will add a comment to this sleep in my pr